src/main/java/com/spotify/docker/client/DockerClient.java

```diff
@@ -604,13 +604,142 @@ public String getName() {
   void close();
 
   /**
-   * Parameters for {@link #logs(String, LogsParameter...)}
+   * Parameters for {@link #logs(String, LogsParam...)}
    */
-  public static enum LogsParameter {
```

Comment:
Ah I see you changed the class name here to LogsParam because can no longer be an enum. And enums in this interface are Parameters while classes are Param. :+1: